components/docker-up/dependencies.sh

```diff
@@ -10,3 +10,4 @@ DOCKER_COMPOSE_VERSION=1.29.2
 
 curl -o docker.tgz      -fsSL https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_VERSION}.tgz
 curl -o docker-compose  -fsSL https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-Linux-x86_64
+curl -o runc            -fsSL https://github.com/opencontainers/runc/releases/download/v1.1.0/runc.amd64
```

Comment:
How about making the version a variable like the others?
```suggestion
curl -o runc            -fsSL https://github.com/opencontainers/runc/releases/download/${RUNC_VERSION}/runc.amd64
```